WeakAuras/Transmission.lua

```diff
@@ -160,86 +160,34 @@ function tableSubtract(minuend, subtrahend)
   recurse(minuend, subtrahend);
 end
 
-function DisplayStub(regionType)
-  local stub = {
-    ["untrigger"] = {
-    },
-    ["animation"] = {
-      ["start"] = {
-        ["duration_type"] = "seconds",
-        ["type"] = "none"
-      },
-      ["main"] = {
-        ["duration_type"] = "seconds",
-        ["type"] = "none"
-      },
-      ["finish"] = {
-        ["duration_type"] = "seconds",
-        ["type"] = "none"
-      }
-    },
-    ["trigger"] = {
-      ["type"] = "aura",
-      ["subeventPrefix"] = "SPELL",
-      ["subeventSuffix"] = "_CAST_START",
-      ["debuffType"] = "HELPFUL",
-      ["names"] = {},
-      ["spellIds"] = {},
-      ["event"] = "Health",
-      ["unit"] = "player"
-    },
-    ["actions"] = {
-      ["init"] = {
-      },
-      ["start"] = {
-      },
-      ["finish"] = {
-      }
-    },
-    ["conditions"] = {
-    },
-    ["load"] = {
-      ["class"] = {
-        ["multi"] = {
-        }
-      },
-      ["spec"] = {
-        ["multi"] = {
-        }
-      },
-      ["size"] = {
-        ["multi"] = {
-        }
-      }
-    }
-  }
 
-  WeakAuras.validate(stub, regionTypes[regionType].default);
-  return stub;
+function GenerateUniqueID()
+  -- generates a unique random 11 digit number in base64
+  -- in principle this could cause collisions if the sun 
+  -- ends but YouTube doesn't seem to be having problems
+  local s = {}
+  for i=1,11 do
+    table.insert(s, bytetoB64[math.random(0, 63)])
+  end
+  return table.concat(s)
 end
 
 function CompressDisplay(data)
-  local copiedData = {};
```

Comment:
Why did you remove the copying before removing custom code?

Only the exported data should be cleaned, the actual auras should not be changed on export.

Reply:
Argh, that's too bad. I guess keep this, even if it's ugly.